plugins/sharding/logical_count.rb

```diff
@@ -57,79 +42,296 @@ def cache_key(input)
       end
 
       class ShardCountContext
```

Comment:
It seems that new `ShardCountContext` is similar to `GroongaSharding::LogicalRangeFilterCommand::ExecuteContext`.
Can we unify common implementation?

Reply:
I have unified common implementation with logical_range_filter in https://github.com/groonga/groonga/pull/1342, https://github.com/groonga/groonga/pull/1343, https://github.com/groonga/groonga/pull/1345.

Reply:
They are merged.